### ISSUE TITLE
Empty Error Messages

### ISSUE THREAD

**Issue by ThomTrab** [linuxmint/mintupdate#657]
**Describe the bug**
When I click the `Refresh` button to update the package's list, I got 2 errors messages, one after the other when clicking `Close`. I'm not able to get ride of them. Neither of them give me useful clue to address the situation.
These messages doesn't prevent the package list to be updated and are not present when using `apt update` from cli

**Screenshots**
First error message
![Capture d’écran du 2021-01-20 18-41-52](https://user-images.githubusercontent.com/9775724/105213700-4f8b3800-5b4f-11eb-978b-c2a82432e086.png)

Second error message
![Capture d’écran du 2021-01-20 18-42-03](https://user-images.githubusercontent.com/9775724/105213717-5619af80-5b4f-11eb-82b8-e96b3efd1486.png)

**To Reproduce**
1. Click on the Refresh button
2. See first error message
3. Click `Close`
4. See second error message

**Expected behavior**
No error messages on `Refresh` button click

**Distribution:**
- [X] Linux Mint
- [ ] LMDE
- [ ] Other (please specify)

**Software version:**
See Help > About in Update Manager.

**Logs:**
Copy & paste the log output from Update Manager > View > Information here.

**Additional context**
Before the issue appeared, I modified the `/etc/apt/preferences` file and finally remove it because it wasn't needed anymore

Thank you for your assistance

**Comment by Hordeking** [comment]
This bug exists in mint 5.7.3 english, as well. It only recently started for me, so I assume that's when this version was rolled out. No error is shown when I run it from a terminal, and no issues are obviously present when I use either Synaptic or apt update.
![Untitled](https://user-images.githubusercontent.com/15934196/110758389-164b7c00-8212-11eb-9cec-a4c5c2c31a3e.png)

I don't think the issue is with Synaptic, despite the modal window having it for a title.

**Comment by mockey** [comment]
I see exactly the same bug for quite some time now. Version 5.7.3 here as well.
`sudo apt update` runs without errors and Reload in Synaptic works as well.
Is there any workaround? Because it seems that no updates are fetched by the Update Manager after this error occurs.

**Comment by Lefuneste83** [comment]
Same extremely annoying issue here for the last 2 days on a freshly upgraded LM 20.2

-I Checked validity of all source files against a working server with same level of LM
-I removed all PPA and additional repos
-I purged then reinstalled mintupdate
-I checked various logs, very little is available for forensics

Exact same symptoms : 
Empty message window first, then second with same messages as above. apt update and apt upgrade both run flawlessly.

I have managed to trace the script causing the problem from /var/log/auth.log : 
/usr/lib/linuxmint/mintUpdate/dpkg_lock_check.sh

Bug cinematic as it is logged : 

Dec  6 20:07:23 plakias sudo: user : TTY=unknown ; PWD=/home/user ; USER=root ; COMMAND=/usr/lib/linuxmint/mintUpdate/dpkg_lock_check.sh
Dec  6 20:07:23 plakias sudo: pam_unix(sudo:session): session opened for user root by (uid=0)
Dec  6 20:07:23 plakias sudo: pam_unix(sudo:session): session closed for user root
Dec  6 20:07:23 plakias sudo: user : TTY=unknown ; PWD=/home/user ; USER=root ; COMMAND=/usr/lib/linuxmint/mintUpdate/dpkg_lock_check.sh
Dec  6 20:07:23 plakias sudo: pam_unix(sudo:session): session opened for user root by (uid=0)
Dec  6 20:07:24 plakias sudo: pam_unix(sudo:session): session closed for user root
Dec  6 20:07:24 plakias sudo: user : TTY=unknown ; PWD=/home/user ; USER=root ; COMMAND=/usr/bin/mint-refresh-cache --use-synaptic 62914568
Dec  6 20:07:24 plakias sudo: pam_unix(sudo:session): session opened for user root by (uid=0)
Dec  6 20:07:32 plakias sudo: pam_unix(sudo:session): session closed for user root

Content of the script is : 
#!/bin/sh
/bin/fuser /var/lib/dpkg/lock

lock file is present and empty  on both systems (Mintupdate OK and KO)

**Comment by Lefuneste83** [comment]
Ok I've found the solution to this crazy problem 

sudo mv /var/lib/synaptic/preferences /var/lib/synaptic/preferences.ORIG

from here : https://github.com/linuxmint/mintupdate/issues/627

**Comment by mockey** [comment]
Amazing! Thanks a lot!

**Comment by ThomTrab** [comment]
Thanks a lot!

**Comment by Hordeking** [comment]
@ThomTrab Before we close it, can we actually see if one of us or the devs wants to implement a code fix for mintupdate? I suspect (but don't know) that the empty prefs file doesn't actually cause a problem for synaptic, since I was using apt and synaptic in parallel, but only had trouble with mintupdate.

**Comment by hamishmb** [comment]
Yep, this shouldn't be closed, because it isn't fixed. This seems to be happening during the automated upgrade between releases for a lot of people. For me it happened after pinning and unpinning a package in Synaptic.

**Comment by clefebvre** [comment]
Duplicate of https://github.com/linuxmint/mintupdate/issues/627
